css/home.css

```diff
@@ -0,0 +1,110 @@
+/* nav container  */
+
+/* .header {
+  height: 15vh;
+} */
+
+.header__nav {
+  background-color: transparent;
+  padding: 1rem;
+  overflow: hidden;
+}
+
+.header__nav--brand {
+  font-size: 1.8rem;
+  font-weight: var(--font-medium);
+  font-family: "Lobster", cursive;
+}
+
+.header__nav--search {
+  position: relative;
+}
+
+.fa-search {
+  position: absolute;
+  right: 0;
+  top: 20%;
+}
+
+.nav__search {
+  width: 400px;
+  padding: 0.5rem;
+  border: none;
+  border-radius: 1px;
+}
+
+.header__nav--menu > li {
```

Comment:
Instead of applying the property to the list elements you can directly apply it to the parent element having the child list elements or even use it as a utility-class list-no-bullets{list-style:none;}

Reply:
Sure, I will be adding more utility classes, this being one of them. Thanks